google_guest_agent/osinfo/osinfo_unix.go

```diff
@@ -130,16 +114,27 @@ func parseID(id string) string {
 	}
 }
 
-func getRelease() (release, error) {
-	if runtime.GOOS == "linux" {
-		releaseFile, err := ioutil.ReadFile("/etc/os-release")
-		if err == nil {
-			return parseOSRelease(string(releaseFile))
-		}
+// Get returns OSInfo on the running system.
+func Get() OSInfo {
+	var osInfo OSInfo
+
+	releaseFile, err := ioutil.ReadFile("/etc/os-release")
```

Comment:
I'd break this block into its own util function as well, receiving a list of possible release file paths.

Reply:
I broke it out, but the paths are specific to the parsing functions.